testing/burp/APKBUILD

```diff
@@ -14,6 +14,8 @@ makedepends="
 	zlib-dev
 	librsync-dev
 	acl-dev
+	libc6-compat
+	check-dev
```

Comment:
If these are purely for check phase you should add them to checkdepends. I do wonder why you need libc6-compat?